docs/cristin.ttl

```diff
@@ -72,6 +72,10 @@
     and they have contributed as "ILLUSTRATOR" affiliated with InstB for the same publication.
     """ .
 
+:VITENSKAPELIGARBEID_LOKAL
```

Comment:
I am not really sure about the meaning of this field.

Reply:
This field contains metadata relevant for NVI reporting, one set per contributing* institution.

So for a given publication/result, let's say there are four authors: One affiliated** with UIB, two affiliated with NTNU and one affiliated with some foreign university.

For the current NVI reporting process, that means that some authority (superuser) from both UIB and NTNU have to verify that this result is worthy of including it in the NVI report, and correctly credited.

The field VITENSKAPELIGARBEID_LOKAL contains the status and verdict for these two institutions. Who verified the process and when, whether they can confirm that some conditions are met, and whether they then regard the result as acceptable for reporting or not.

In terms of data modeling, this information has a different cardinality/multiplicity than the authors or their affiliations, so it is stored in a separate array of objects.

*) only "cristin institutions", i.e. those that participate in the NVIU process, get a record in the VITENSKAPELIGARBEID_LOKAL table
**) technically it's about the unique set of insitutions representing the different and overlapping affiliations that the different authors may have